source/features/github-bugs.css

```diff
@@ -77,3 +77,19 @@
 .pagehead-actions :is(.btn, summary) .octicon {
 	margin-right: 4px !important;
 }
+
+/* Avoid excessive repo title wrap by deprioritizing the repo header actions #5398 */
+:root .pagehead-actions {
+	flex-shrink: 3 !important;
+	display: inline-flex !important;
+	flex-wrap: wrap;
+	justify-content: flex-end;
+}
+
+:root .pagehead-actions > li {
```

Comment:
On non-personal repos there's an empty element (Edit pins) that starts taking up space once we add `margin-bottom`